src/ui/components/component.js

```diff
@@ -109,10 +109,11 @@ export default class Component {
     }
 
     /**
-     * A custom class to be applied to {this._container} node
+     * A custom class to be applied to {this._container} node. Note that the class
+     * 'component' will be included also.
```

Comment:
tiny nit: can we reword "will be included also" to "will also be included"?

Reply:
or "will also be applied"

Reply:
If this wording really bothers you, I can change. But, given how close the two phrases are, I'd rather keep as is and not make another commit.

Reply:
It's totally fine as it is, haha